src/Sarif.UnitTests/TestData/InsertOptionalDataVisitor/CoreTests.sarif

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "http://json.schemastore.org/sarif-2.0.0",
+  "version": "2.0.0",
+  "runs": [
+    {
+      "tool": { "name": "Sarif.UnitTests",
+        "fullName": "Sarif.UnitTests.1.3.3.0-beta", "version": "1.3.3.0", "semanticVersion": "1.3.3", "sarifLoggerVersion": "1.5.22.0", "language": "en-US"},
+      "originalUriBaseIds": { "SRCROOT": "file:///e:/src/sarif-sdk/" },
```

Comment:
>e [](start = 50, length = 1)

Do you have an E: drive? Is that why the tests pass for you but not on AppVeyor?

Reply:
Nerts. of course.

---
In reply to: [205980932](https://github.com/Microsoft/sarif-sdk/pull/974#discussion_r205980932) [](ancestors = 205980932)